platform/android/export/export.cpp

```diff
@@ -1381,7 +1381,7 @@ Error EditorExportPlatformAndroid::export_project(const String& p_path, bool p_d
 
 		err = OS::get_singleton()->execute(jarsigner,args,true,NULL,NULL,&retval);
 		if (retval) {
-			EditorNode::add_io_error("'jarsigner' verificaiton of APK failed. Make sure to use jarsigner from Java 6.");
+			EditorNode::add_io_error("'jarsigner' verification of APK failed. Make sure to use jarsigner from Java 6.");
```

Comment:
I take note. Since that typo was already there I thought I had to choose between a PR only for that or at least a separate commit.

Once I fixed #5208 I'm fusing all commits in this PR to only one.